### PR TITLE
Add no cluster error handling

### PR DESCRIPTION
**Pull request by PiekarzKarolina**

### CODE REVIEW

File: gui/functionality/add_cluster.py [hunk 1]
```diff
@@ -54,7 +55,9 @@ def handle_complete_add_cluster(self):
         self.popup_window.show_message("The cluster has been successfully added", WindowType.success)
 
     def handle_error_add_cluster(self, exception):
-        if isinstance(exception, ValueError):
+        if isinstance(exception, EmptyFieldError):
```
Comment (pswedrak) [inline gui/functionality/add_cluster.py]: How can a cluster name be empty?

Comment (PiekarzKarolina) [reply gui/functionality/add_cluster.py]: Becasue somebody can just try to break our application :P

Comment (pswedrak) [reply gui/functionality/add_cluster.py]: Is it possible to add a cluster with an empty name?

Comment (PiekarzKarolina) [reply gui/functionality/add_cluster.py]: No but the error for this was just an empty string. So I decided to make it more verbose.

Comment (pswedrak) [reply gui/functionality/add_cluster.py]: So when can this error happen if there is no cluster with an empty name?

Comment (PiekarzKarolina) [reply gui/functionality/add_cluster.py]: It's in adding cluster. It doesn't have the clusters list. Just remove the name.

Comment (pswedrak) [reply gui/functionality/add_cluster.py]: ok

Comment (PiekarzKarolina) [reply gui/functionality/add_cluster.py]: Plus if it would be a cluster list, if you don't have any clusters the story is the same

**Pull request merged**